pages/linux/dex.md

```diff
@@ -0,0 +1,32 @@
+# dex
+
+> DesktopEntry Execution is a program to generate and execute DesktopEntry files of the Application type.
+> More information: <https://github.com/jceb/dex>.
+
+- Perform an autostart/execute all programs in the autostart folders:
+
+`dex --autostart`
+
+- Perform an autostart/execute all programs in the specified folders:
```

Comment:
```suggestion
- Execute all programs in the specified folders:
```
Same here

Will this command execute any DesktopEntries it finds in those folders?

Reply:
Copied over from `dex` docs/examples, so whatever you wish.